### PR TITLE
Move orbital selection to arguments for GreenFunction etc.; remove DensityMatrix.

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: include/edlib/ChiLoc.h [hunk 1]
```diff
@@ -93,6 +93,9 @@ namespace EDLib {
     /**
      * General class for local susceptibilities calculation.
      *
+     * By default, all local Green functions are computed. An array of orbital
```
Comment (iskakoff) [inline include/edlib/ChiLoc.h:96]: We basically have duplicated functionality here and in GreensFunction.h

I propose to move body of the constructor into base class function and, If you really want to have different off-diagonal orbitals for Chi and for G, pass group name as a parameter.

**Comment by ghost** [comment]
Well, I was asked to compute them separately and want to be able to do it without kludges.

File: include/edlib/ChiLoc.h (hunk 1 above)
Comment (ghost) [reply include/edlib/ChiLoc.h:96]: I will try to.

**Comment by iskakoff** [comment]
Great, and now we do not have G in output anymore. 
What you should be doing is the following:
1. Compute greater and lesser G
2. Sum it together and store in hdf5 and text file the normal G
3. Provide access to G_g, G_l through getters
4. In your executable call this getters and do whatever you want to do

Because:
1. It does not break functionality (I do not need to have G_l and G_g in my code I want to have G, which I do not have anymore)
2. It will provide flexibility so G_l and G_g can be accessed for some test purpose or whatever

**Pull request merged**